client/components/Pager.jsx

```diff
@@ -1,19 +1,132 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import ReviewList from './ReviewList';
+
+// TODO: Refacor styling
+const List = styled.ul`
+  list-style: none;
+  padding: 6px;
+  height: 36px;
+`;
+
+const ListItem = styled.li`
+  margin-right: 6px;
+  display: inline-block;
+  text-align: center;
+  font-size: 13px;
+  font-weight: 700;
+  line-height: 36px;
+  width: 36px;
+  height: 36px;
+  border-radius: 18px;
+  background-color: #efefef;
+  color: ${(props) => props.theme.colors.paleGray};
+`;
+
+const Link = styled.a`
+  &:hover {
+    background-color: #dedede;
+  };
+  &.active {
+    background-color: #a7a7a7;
+  }
+  width: 100%;
+  height: 100%;
+  border-radius: 18px;
+  display: inline-block;
+  text-decoration: none;
+  color: ${(props) => props.theme.colors.darkGray};
+`;
+
+const Span = styled.span`
+  width: 18px;
+  height: 18px;
+  display: inline-block;
+  vertical-align: sub;
+`;
+
+// conditonal rendering using built-in "children" prop:
+// if condition true rander element with link around it otherwise without link
+function linkIf(condition, props, children) {
+  const { onClick, href = '#' } = props;
+  if (condition) {
+    return <Link onClick={onClick} href={href}>{children}</Link>;
+  }
+  return children;
+}
 
 class Pager extends React.Component {
   render() {
+    const pages = [];
+    const { props } = this;
+    for (let i = 1; i < props.totalPages + 1; i += 1) {
```

Comment:
Did you mean this to be i+= instead of i++?

Reply:
Eslint 🙄  😅